config-sample.ini

```diff
@@ -28,6 +28,15 @@ keyPath=
 # expressjs shortcuts are supported: loopback(127.0.0.1/8, ::1/128), linklocal(169.254.0.0/16, fe80::/10), uniquelocal(10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16, fc00::/7)
 trustedReverseProxy=false
 
+
+[Cookies]
```

Comment:
I think it would make more sense to have `Session` instead of `Cookies`? Since we are explicitly configuring the session cookie and not other potential cookies we might have (for example the one which stores if we are on mobile or desktop view).

Reply:
I suppose you're right (althought *technically* I will need to use the same path for the csrf cookie, but that also kinda has something to do with sessions ;-) ), will change accordingly :-)